submissions/the-darkesttt/frogger-task/app.js

```diff
@@ -0,0 +1,323 @@
+const playerSkins = [
+    {
+        id: 'boy',
+        src: 'images/char-boy.png',
+    },
+    {
+        id: 'cat-girl',
+        src: 'images/char-cat-girl.png',
+    },
+    {
+        id: 'horn-girl',
+        src: 'images/char-horn-girl.png',
+    },
+    {
+        id: 'pink-girl',
+        src: 'images/char-pink-girl.png',
+    },
+    {
+        id: 'princess-girl',
+        src: 'images/char-princess-girl.png',
+    },
+];
+
+class Menu {
+    bg;
```

Comment:
It is not required to define properties like that before the constructor. They will be created there.